axelrod/fingerprint.py

```diff
@@ -37,14 +37,15 @@ def create_points(step):
 
 
 class AshlockFingerprint():
-    def __init__(self, strategy, probe):
+    def __init__(self, strategy, probe=axl.TitForTat):
         """
         Parameters
         ----------
         strategy : class
             A class that must be descended from axelrod.Player
         probe : class
             A class that must be descended from axelrod.Player
```

Comment:
Technically it could but we're using decorators (which require a class) down the line and that would make things slightly more complicated than I intended with this PR. I'll stop being lazy and do it...